content/sensu-go/5.14/guides/clustering.md

```diff
@@ -158,6 +158,25 @@ ETCD_INITIAL_CLUSTER="backend-4=https://10.0.0.4:2380,backend-1=https://10.0.0.1
 ETCD_INITIAL_CLUSTER_STATE="existing"
 {{< /highlight >}}
 
+#### Replace a faulty cluster member
+
+Run [`sensuctl cluster health`][17] to check for faulty cluster members. For faulty cluster members, the `Error` column will include an error message and the `Healthy` column will list `false`. Replace faulty cluster members to restore the cluster's health. 
+
+To replace a faulty cluster member, make sure to change the `etcd-initial-cluster-state` to `existing` in the configuration. This example store configuration shows `backend-3` as the faulty cluster.
```

Comment:
So we'll need to add some commands above on how to remove the faulty member as well, as this section is all about how to replace a fault member.

This page already has those commands above. We'll probably want to show a faulty member using `sensuctl cluster member-list`, then the command to remove it `sensuctl cluster member-remove 2f7ae42c315f8c2d`. From there, we can talk about adding the new one and call out that the below configuration is for the new member only.

We can pair up and go over this more a bit later if needed with 5.15 the priority at the moment

Reply:
@rgeniesse your explanation makes sense. Let me try to fix it and see what you think. Thank you!